### ISSUE TITLE
Browserify support removed in jsdom 22.0.0

### ISSUE THREAD

**Issue by steffen-stell** [nanxstats/r-readability-parser#1]
Hey,
thank you for sharing your tutorial. Unfortunately, jsdom removed support for browserify in [version 22.0.0](https://github.com/jsdom/jsdom/releases/tag/22.0.0). Is there any chance you'll update it to replace jsdom?

**Comment by nanxstats** [comment]
Great question, but not sure if I know the best solution for this particular issue.

However, here is a simple solution: use an older version of jsdom (20.0.0). I tested it and it works with the current versions of Browserify, DOMPurify, and Readability.js. I've updated the [blog post](https://nanx.me/blog/post/r-readability-parser/) to document this.

The JS libraries snapshot in this repo still works.

**Comment by steffen-stell** [comment]
I don't think this is a good idea. The [DOMPurify documentation](https://github.com/cure53/DOMPurify#running-dompurify-on-the-server) is quite explicit about the need of an up-to-date version of jsdom, because XSS vulnerabilities in jsdom can and do occur and get fixed in newer releases. This is why I did not just take the browserified scripts from the repo. 

When I find the time, I'll look into passing the html from R to node.js and avoiding browserify that way. I'll share it here if I get something usable.
